test/ia-book-actions.test.js

```diff
@@ -445,7 +445,7 @@ describe('Auto renew one hour loan', () => {
     expect(el.loanRenewResult.renewNow).to.equal(true);
     expect(el.loanRenewResult.secondsLeft).to.equal(10);
     expect(el.timerCountdownEl).to.exist;
-    expect(el.timerCountdownEl.secondsLeftOnLoan).to.equal(loanTime);
+    expect(el.timerCountdownEl.secondsLeftOnLoan).to.equal(8);
```

Comment:
Q: why did we change this?

Reply:
we have changed the secondsLeftOnLoan key when loan was renewed so new updated time needs to be there.

Reply:
Yes